lib/loader/ast/instruction.cpp

```diff
@@ -313,8 +313,13 @@ Expect<void> Loader::loadInstruction(AST::Instruction &Instr) {
     if (auto Res = readU32(Instr.getMemoryAlign()); unlikely(!Res)) {
       return Unexpect(Res);
     }
-    if (Conf.hasProposal(Proposal::MultiMemories) &&
-        Instr.getMemoryAlign() >= 64) {
+    if (Instr.getMemoryAlign() >= 128 ||
```

Comment:
The following is better:

```
if (Align >= 128) {
  ... error
} else if (Align >= 64) {
  if (hasMultiMemories) {
    ... read memory index
  } else {
    ... error
  }
}
```

Reply:
but clang suggested do not use 'else' after 'return' 🤔

Reply:
That's ok, we don't follow that rule.